docs/guide/uploading-downloading-binary-content.md

```diff
@@ -0,0 +1,147 @@
+# Uploading & Downloading Binary Content | AngularFire Guide
+
+## Table of Contents
+
+* [Overview](#overview)
+* [API Summary](#api-summary)
+* [Uploading files](#uploading-files)
+* [Displaying images with the `firebase-src` directive](#displaying-images-with-the-firebase-src-directive)
+* [Retrieving files from the template](#retrieving-files-from-the-template)
+
+## Overview
+
+Firebase provides [a hosted binary storage service](https://firebase.google.com/docs/storage/)
+which enables you to store and retrieve user-generated content like images, audio and
+video directly from the Firebase client SDK.
+
+Binary files are stored in a Firebase Storage bucket, not in the Realtime Database.
+The file files in your bucket are stored in a hierarchical structure, just like
+in the Realtime Database.
+
+To use the Firebase Storage binding, first [create a Firebase Storage reference](https://firebase.google.com/docs/storage/web/create-reference).
+Then using this reference, pass it into the `$firebaseStorage` service.
+
+```js
+// define our app and dependencies (remember to include firebase!)
+angular
+  .module("sampleApp", [
+    "firebase"
+  ])
+  .controller("SampleCtrl", SampleCtrl);
+
+// inject $firebaseStorage into our controller
+function SampleCtrl($firebaseStorage) {
+  // create a Firebase Storage Reference for the $firebaseStorage binding
+  var storageRef = firebase.storage().ref('userProfiles/physicsmarie');
+  var storage = $firebaseStorage(storageRef);
+}
+SampleCtrl.$inject = ["$firebaseStorage"];
```

Comment:
Do we actually need this line? Also, can we make this style consistent with how we define the module and controller elsewhere in the docs (e.g. [here](https://github.com/firebase/angularfire/blob/d4fc74fe0dcebe643e9a8eaa727ebdb149f984a1/docs/guide/synchronized-arrays.md#overview))?